### DISCUSSION TITLE
Count the occurences of a given field using the SEARCH method

### DISCUSSION THREAD

**Discussion by SavArmadillo** [directus/directus#13827] [Archive]
Hello!

I've been playing around the Directus api for quite a while now but i'm stuck on a problem i'm not even sure it's possible to resolve it.

Let's say I have a series of items with the field `X` that can only have `a`, `b` or `c` as it's value.

For example, taking a sample of 10 items, I have 3 a 1 b and 6 c

Is there a way that I can count them using the api parameters with the SEARCH method?

So far, the closest I've been is by usig the aggregation with count or countDistinct but they only return me the total number of data the field have.

**Comment by rijkvanzanten** [comment]
I think you'll get the results you're looking for if you use Count() combined with the Group on the field X 🙂 

So `aggregate[count]=id&groupBy=X`

**Reply by SavArmadillo** [reply]
It seems that is exactly what I have been looking for, thanks!
